src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/BungeeManager.java

```diff
@@ -0,0 +1,274 @@
+package net.sacredlabyrinth.phaed.simpleclans.managers;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import net.sacredlabyrinth.phaed.simpleclans.Clan;
+import net.sacredlabyrinth.phaed.simpleclans.ClanPlayer;
+import net.sacredlabyrinth.phaed.simpleclans.SimpleClans;
+import net.sacredlabyrinth.phaed.simpleclans.managers.SettingsManager.ConfigField;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.messaging.Messenger;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Level;
+
+import static net.sacredlabyrinth.phaed.simpleclans.SimpleClans.debug;
+
+public final class BungeeManager {
```

Comment:
if you're going to implement Velocity support, it would be a good idea to make an abstract of this. 
||Said making it final at first, suggesting making an abstract at second 👍🏻 ||